nixos/modules/services/backup/znapzend.nix

```diff
@@ -0,0 +1,36 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.znapzend;
+in
+{
+  options = {
+    services.znapzend = {
+      enable = mkEnableOption "Enable ZnapZend daemon";
```

Comment:
The "Enable" here is redundant, `mkEnableOption` creates a description for you; typically you'll just pass in the name of the service here.